src/components/common/Extensions/ImageCarousel/ImageCarousel.tsx

```diff
@@ -14,7 +14,7 @@ const ImageCarousel: FC<ImageCarouselProps> = ({
 }) => {
   const windowSize = useWindowSize();
   const width = windowSize?.width ?? 0;
-  const setSlidePercentage = (width >= 1440 && 89) || (width >= 850 && 60);
+  const setSlidePercentage = (width >= 1440 && 89.1625) || (width >= 850 && 60);
```

Comment:
this value is weird

Reply:
I know about it but the client has to have exactly the proper size of the image (more details: https://tw.auroracreation.com/app/tasks/15516036).
Also, I consider a change the slide plugin because this one doesn't work correctly in this slider, but I would do it in a separate PR.